### ISSUE TITLE
Cannot define quota limit for blobstore

### ISSUE THREAD

**Issue by android-leha** [datadrivers/terraform-provider-nexus#41]
I have in main.tf
```
soft_quota {
     limit = 1024
     type = "spaceUsedQuota"
}
```
But getting in UI:
![image](https://user-images.githubusercontent.com/1458524/82144291-715d4e00-984f-11ea-9ca9-fb2073c01ae7.png)

nexus version: 3.23.0-03

Update: Same for "spaceRemainingQuota"

**Comment by Nosmoht** [comment]
Hi @android-leha ,

which version of the provider are u using?

**Comment by Nosmoht** [comment]
Hi @android-leha,

missing repository update was fixed with https://github.com/datadrivers/terraform-provider-nexus/commit/94b3e77a8c2a848940d398e833eea72bcbc0f46b.
Could you please test with current release [v1.6.0](https://github.com/datadrivers/terraform-provider-nexus/releases/tag/v1.6.0).

**Comment by android-leha** [comment]
I checked it with latest version from master branch(I think it was 1.5.3) and also rechecked it with 1.6.0 and latest from today (fe6d754abde5b2b18b0c003f8e15ff1e5602690e)

All the same

**Comment by android-leha** [comment]
Hi,
I checked this api directly and it is not working as well.
`
curl -X PUT "http://localhost:8081/service/rest/beta/blobstores/file/default" -H "accept: application/json" -H "Content-Type: application/json" -d "{ \"softQuota\": { \"type\": \"spaceRemainingQuota\", \"limit\": 2024 }, \"path\": \"default\"}"
`

Looks like bug in nexus itself

**Comment by android-leha** [comment]
Hi @Nosmoht 

I opened an issue in nexus Jira https://issues.sonatype.org/browse/NEXUS-23987 and got answer that problem that limit should be defined in bytes.
In parallel another ticket was opened for this: https://issues.sonatype.org/browse/NEXUS-23985

So need or multiple current value to 100000 or update your examples with right values, or just wait for fix=)

**Comment by anmoel** [comment]
Hi @android-leha,

thanks for your report.
I can recreate this problem and will fix it in the next few weeks.

regards
André
